lib/common-utils.coffee

```diff
@@ -0,0 +1,25 @@
+debug = require('debug')('sphere-product-import-common-utils')
+_ = require 'underscore'
+_.mixin require 'underscore-mixins'
+
+class CommonUtils
```

Comment:
There will be many more methods here... will refactor the product-import. This is a place holder for them :)